### PR TITLE
Change network interface to Mosquitto

### LINKED ISSUE #30: Change network interface to mosquitto service
**Describe the bug**
When execute the integration tests, the are conection refused, becouse mosquitto service is running in a difference network interface to 127.0.0.1.

**To Reproduce**
1. Go to tests directory
2. Execute the integration test
4. See error, connection refused.

**Expected behavior**
The integration tests executed successfully.

**Additional context**
The ip address of mosquitto currently is 172.26.0.1 in the port 1883.

### PR DESCRIPTION
**Pull request by ingluife**
Hi, @flyve-mdm/backend-development.

configure mosquitto in the host machine network

### Changes description

I changed the network interface where running mosquitto service. This change derived of the connection refused in the execution of integration test.

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@ingluife |11|

### References

Closes #30

### CODE REVIEW

File: docker/php/Dockerfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ RUN apt-get install -y nodejs \
 # Install iconv, mcryot, gd
 RUN apt-get install -y \
         cron \
+        telnet \
```
Comment (btry) [inline docker/php/Dockerfile:55]: telnet  ??? I would prefer ssh !

**Review by ajsb85 — DISMISSED** [review]

File: docker/php/Dockerfile (hunk 1 above)
Comment (ingluife) [reply docker/php/Dockerfile:55]: I do use it for test the port connection.

**Comment by ingluife** [comment]
Hi, @flyve-mdm/backend-development.

I have a issue in the connect to mosquitto container. I am trying use iptables for do forward the request from 127.0.0.1:1883 to the msoquitto service.

**Comment by btry** [comment]
@ingluife you do things with iptables because unit tests limit Mosquitto to 127.0.0.1 ?

why not modify the unit tests to listen to an other interface ?

I suggest we add configuration settings in the CLI installer of the plugin to set IP and port for mqt broker. This way we can remove the hard coded settings in the unit tests and move them in the shell scripts for Travis.

For your containers you will be able to use alsi the cli installer to configure at your convenience the IP address of the broker.

**Comment by ingluife** [comment]
I dismissed the use of iptables. The reason is in this reference:
https://github.com/flyve-mdm/glpi-plugin/pull/577

**Review by ajsb85 — DISMISSED** [review]

**Comment by ingluife** [comment]
Hi @flyve-mdm/backend-development. 

This is ready for merge.

**Review by btry — APPROVED** [review]

**Pull request merged**